### DISCUSSION TITLE
我在内部调用方法总是出问题呢

### DISCUSSION THREAD

**Discussion by wanfengok** [zouyaoji/vue-cesium#198] [Q&A] [answered]
组件内部写函数总是找不到此方法。
![image](https://user-images.githubusercontent.com/22619704/165872789-8e2f33ec-3887-4cb2-b34b-78813e921122.png)

**Comment by zouyaoji** [comment]
这似乎与vue-cesium无关吧

**Comment by wanfengok** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/22619704/165873583-f6e77b5c-09c8-4e7c-b874-021d6d0f6497.png)

**Reply by wanfengok** [reply]
就是里面写上任何方法总是调用失败。。 顺便问下。线有贴地的模型 那个文字和点 似乎好像不支持模型贴地。

**Reply by zouyaoji** [reply]
这种应该是写法问题， 你看看vue的文档哈。
贴地那个只有线面直接支持， 文字点不行哈，没有高度的话要自己采样获取高度。

**Reply by wanfengok** [reply]
如果能加进去就更好拉 。我也先找找的。你的框架非常不错，希望越做越好。

**Comment by wanfengok** [comment]
找了一段代码 不知道能不能用到这个点和字体的显示

5## 根据坐标获取 terrain 高程，精度为 m

var terrainData = new Cesium.CesiumTerrainProvider({

url:this.server_url_outer+"terrain/" //输入 terrian 地形数据加载地址

});

var positions = [

Cesium.Cartographic.fromDegrees(lon,lat) //输入经纬度

];

var promise = Cesium.sampleTerrain(terrainData,14,positions); //获取 14 级地形高程

Cesium.when(promise, function (updatedPositions) {

var terrainHeight = updatedPositions[0].height;

});``

**Reply by zouyaoji** [reply]
可以哈，这个就是基于地形采样的。

**Reply by wanfengok** [reply]
写成贴点和贴标签的插件吧 嘻嘻 相信有很多人愿意用哈
